engine/src/main/java/org/camunda/bpm/engine/impl/batch/AbstractBatchJobHandler.java

```diff
@@ -137,5 +194,14 @@ public T readConfiguration(byte[] serializedConfiguration) {
     return getJsonConverterInstance().toObject(JsonUtil.asObject(serializedConfiguration));
   }
 
+  protected void createSingleDeploymentIdMappingForDefinition(T configuration, String processDefinitionId) {
```

Comment:
Method is never called.

Reply:
Another candidate of being missed to be removed after refactoring the backwards compatibility implementation, thanks!